build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-beta4'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta6'
```

Comment:
Do we need to use the beta version?

Reply:
yes, do yo ask the same in the previous PR.

Reply:
I don't remember why it's needed, but don't worry. I'll review the previous PR.